catalog/async/catalog-async-data-api/src/main/java/org/codice/ddf/catalog/async/data/api/internal/ProcessResource.java

```diff
@@ -87,4 +87,11 @@
      * @return {@code true} if modified, {@code false} otherwise
      */
     boolean isModified();
+
+    /**
+     * This is used to close the input stream.
+     * <p>
+     * Look into having the ProcessResource implement InputStream
```

Comment:
Is `Look into having the ``ProcessResource implement InputStream` still a TODO? Can that message be removed? Would it be easy to have` ProcessResource` extend `java.io.closable`? Would it provide value?

Reply:
Yes, this is still a TODO.
There was talk about maybe re implementing the process resource stuff.

Created a ticket for this:
https://codice.atlassian.net/browse/DDF-3389